src/transformers/models/gpt_neox/tokenization_gpt_neox_fast.py

```diff
@@ -105,8 +105,8 @@ def __init__(
         **kwargs,
     ):
         super().__init__(
-            vocab_file,
-            merges_file,
+            vocab_file=vocab_file,
+            merges_file=merges_file,
```

Comment:
why did you add that ?

Reply:
Because of tokenization_utils_fast.py (L120-123). If we do not explicitly assign vocab_file and merges_file params just like that, `kwargs.get("vocab_file")` returns None and `gguf_param = load_gguf_checkpoint(kwargs.get("vocab_file"))` will not work. The same things we did for Bloom and Qwen